modules/tensor_mechanics/src/materials/ADComputeMultipleInelasticStress.C

```diff
@@ -390,6 +390,9 @@ ADComputeMultipleInelasticStress::computeAdmissibleState(
     ADRankTwoTensor & elastic_strain_increment,
     ADRankTwoTensor & inelastic_strain_increment)
 {
+  // Properly update material properties (necessary if substepping is employed).
+  _models[model_number]->zeroOutIncrementalMaterialProperties();
```

Comment:
if it is only necessary when substepping is employed, maybe you could add 
```
if (_use_substep)
```

Reply:
This method will provide the value of material properties (which will be zero or "reset") if substepping is not selected. Creep models that include substepping will make use of this method, even if the user has not selected it.